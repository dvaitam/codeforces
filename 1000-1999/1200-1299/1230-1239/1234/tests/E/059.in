18 6
1 18 5 17 14 5
