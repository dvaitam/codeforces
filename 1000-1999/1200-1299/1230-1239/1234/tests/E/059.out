54 53 53 53 65 50 50 50 50 50 50 50 50 60 50 50 56 30
